pkg/aws/helpers.go

```diff
@@ -106,6 +108,17 @@ func GetRegion(region string) (string, error) {
 
 		region = *defaultSession.Config.Region
 	}
+	VPCRegions := awsenum.VPCRegion.Values("")
```

Comment:
Not sure if using this is the best path https://github.com/aws/aws-sdk-go-v2/blob/main/service/route53/types/enums.go#L535

But it covers all current regions, so it is better than not validating or having a specific set of regions to validate against maintained on the CLI

What do you think about it?

Reply:
We have an endpoint in OCM to get all the available supported region per user account:
`ocm post /api/clusters_mgmt/v1/aws_inquiries/regions`
We use it right after calling this function `GetRegion`.

I'm not sure if we support all regions retrieved from `awsenum.VPCRegion.Values("")`.